src/main/java/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,43 @@
+package dao;
+
+import db.UserStorage;
+import exceptions.WrongPasswordException;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import model.Bet;
+import model.User;
+
+public class UserDaoImpl implements UserDao {
+    private Map<String, User> storage = UserStorage.userStorage;
```

Comment:
Do you real should use Map and userName as key? User object already contains userName value

Reply:
done